gammapy/irf/psf_king.py

```diff
@@ -92,18 +92,18 @@ def from_table(cls, table):
         table : `~astropy.table.Table`
             Table King PSF info.
         """
-        offset_lo = table['THETA_LO'].squeeze()
-        offset_hi = table['THETA_HI'].squeeze()
+        offset_lo = table['THETA_LO'].quantity[0]
+        offset_hi = table['THETA_HI'].quantity[0]
         offset = (offset_hi + offset_lo) / 2
         offset = Angle(offset, unit=table['THETA_LO'].unit)
 
-        energy_lo = table['ENERG_LO'].squeeze()
-        energy_hi = table['ENERG_HI'].squeeze()
+        energy_lo = table['ENERG_LO'].quantity[0]
+        energy_hi = table['ENERG_HI'].quantity[0]
         energy_lo = Energy(energy_lo, unit=table['ENERG_LO'].unit)
         energy_hi = Energy(energy_hi, unit=table['ENERG_HI'].unit)
 
-        gamma = Quantity(table['GAMMA'].squeeze(), table['GAMMA'].unit)
-        sigma = Quantity(table['SIGMA'].squeeze(), table['SIGMA'].unit)
+        gamma = Quantity(table['GAMMA'].quantity[0], table['GAMMA'].unit)
```

Comment:
@thomasarmstrong - You have this:
```
gamma = Quantity(table['GAMMA'].quantity[0], table['GAMMA'].unit)
```
This should be the same, just passing through the Quantity constructor one times less:
```
gamma = table['GAMMA'].quantity[0]
```
Can you please change here and review the diff and also do it in the other cases, at least where it's clearly the same. (changing e.g. from `Angle` to `Quantity` might break callers, altough for uniformity we probably should change to `Quantity` everywhere. (`Angle` is a `Quantity` subclass with a little sugar.